common/lib/dependabot/pull_request_updater/github.rb

```diff
@@ -173,7 +173,8 @@ def update_branch(commit)
 
         if e.message.match?(/protected branch/i) ||
            e.message.match?(/not authorized to push/i) ||
-           e.message.match?(/must not contain merge commits/)
+           e.message.match?(/must not contain merge commits/) ||
+           e.message.match?(/required status check/i)
```

Comment:
There are a few variations of the error message, depending on wether there is 1 required check or more enabled, it didn't seem pertinent to test every variation, but that's why the check is not case sensitive